db/re/item_db_usable.yml

```diff
@@ -47391,6 +47391,139 @@ Body:
     Weight: 50
     Script: |
       laphine_upgrade();
+  - Id: 23505
+    AegisName: Yellow_Potion_B_20
+    Name: "[Not for Sale] Yellow Potions Box 20"   # !todo check english name
+    Type: Usable
+    Trade:
+      NoDrop: true
+      NoTrade: true
+      NoCart: true
+      NoStorage: true
+      NoGuildStorage: true
+      NoAuction: true
+    Script: |
+      getitem 11566,20;
+  - Id: 23506
+    AegisName: White_Potion_B_20
+    Name: "[Not for Sale] White Potion Box 20"   # !todo check english name
+    Type: Usable
+    Trade:
+      NoDrop: true
+      NoTrade: true
+      NoCart: true
+      NoStorage: true
+      NoGuildStorage: true
+      NoAuction: true
+    Script: |
+      getitem 504,20;
```

Comment:
https://divine-pride.net/database/item/23506 according to DP is ID 504

Reply:
`[비매품] 하얀포션` according to the description. I doubt the description is wrong in this case since `[Not for Sale] Yellow Potions Box 20` is giving `[비매품] 노란포션` ([Event] Yellow Potion)

Reply:
Ah okay thanks